### PR TITLE
🐛 Fixes for protocol & new apb format

### PR DESCRIPTION
**Pull request by david-martin**
* Uses http format for 'uri' value in secret i.e. what the 'Dashboard URL' in MCP will be
* Specify a version
* Use namespace when creating all resources

### CODE REVIEW

File: apb.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+version: 1.0
 name: keycloak-apb
 image: feedhenry/keycloak-apb
```
Comment (matzew) [inline apb.yml]: the `image:` is gone in the `1.0` version format

File: roles/provision-keycloak-apb/templates/secret.yml.j2 [hunk 2]
```diff
@@ -9,7 +9,7 @@ stringData:
   name: keycloak
   admin_username: {{ ADMIN_NAME }}
   admin_password: {{ ADMIN_PASSWORD }}
-  uri: https://{{ keycloak_route.stdout }}
+  uri: http://{{ keycloak_route.stdout }}
```
Comment (matzew) [inline roles/provision-keycloak-apb/templates/secret.yml.j2]: wondering, why http ? 
does that require extra setup of Keycloak (e.g. SSL disabled?)

File: apb.yml (hunk 1 above)
Comment (philbrookes) [reply apb.yml]: oh, this is great news!

File: roles/provision-keycloak-apb/templates/secret.yml.j2 (hunk 2 above)
Comment (david-martin) [reply roles/provision-keycloak-apb/templates/secret.yml.j2]: @matzew I don't know why it was http. Maybe @philbrookes or @maleck13 know?
I'm sure we can change it

File: apb.yml (hunk 1 above)
Comment (david-martin) [reply apb.yml]: will remove :+1:

Comment (david-martin) [reply apb.yml]: @matzew @philbrookes is there an apb validation thing we could use for CI?

**Comment by david-martin** [comment]
Merging with follow up issue created for switching to https https://issues.jboss.org/browse/FH-4333

**Pull request merged**